src/material-experimental/mdc-slider/testing/slider-harness.ts

```diff
@@ -6,9 +6,63 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ComponentHarness} from '@angular/cdk/testing';
+import {ComponentHarness, HarnessPredicate} from '@angular/cdk/testing';
+import {coerceNumberProperty} from '@angular/cdk/coercion';
+import {SliderHarnessFilters, ThumbPosition} from './slider-harness-filters';
+import {MatSliderThumbHarness} from './slider-thumb-harness';
 
 /** Harness for interacting with a MDC mat-slider in tests. */
 export class MatSliderHarness extends ComponentHarness {
-  // TODO(wagnermaciel): Implement this in a separate PR
+  static hostSelector = '.mat-mdc-slider';
```

Comment:
I was a little on the fence on whether to implement separate harnesses for range and non-range sliders since the non-range is the most common case and having one harness means that the consumer would have to go through `getStartThumb` all the time. I decided against it in order to match what the component's API looks like, but I'm open to changing it. If consumers find it annoying, they could query for `MatSliderThumbHarness` directly as well.